source/ns_cmdline.c

```diff
@@ -126,6 +126,12 @@
 // end of default configurations
 #endif
 
+#if defined(__GNUC__) || defined(__clang__) || defined(__CC_ARM)
+#define CMDLINE_UNUSED __attribute__((unused))
```

Comment:
https://github.com/ARMmbed/mbed-os/blob/a23a850a422e23531a260dd829bce9fd58aa9adf/platform/mbed_toolchain.h#L95 introduce similar attribute I wanted to keep cli as independent as possible and not rely on mbed-os macros. Not sure about what is differences between `__attribute__((unused))` and `__attribute__((__unused__))`. First one is documented [here](https://gcc.gnu.org/onlinedocs/gcc-3.2/gcc/Variable-Attributes.html).

Reply:
@kjbracey-arm do you know difference? Also looks that this did not work for CI runs even it worked in my machine.🤔 needs more investigation..

Reply:
I think I found the reason..

Reply:
Fo